diff -Nur ../lockfile-progs-0.1.10/debian/changelog ./debian/changelog
--- ../lockfile-progs-0.1.10/debian/changelog	2004-02-28 10:44:30.000000000 -0700
+++ ./debian/changelog	2005-09-30 14:48:52.000000000 -0600
@@ -1,3 +1,10 @@
+lockfile-progs (0.1.11) unstable; urgency=low
+
+  * applied patch to fix confusion about --retry. (closes: #161685, #244314)
+    Thanks Dave Anselmi.
+
+ -- Rob Browning <rlb@defaultvalue.org>  Fri, 30 Sep 2005 11:44:30 -0600
+
 lockfile-progs (0.1.10) unstable; urgency=low
 
   * update manpage to fix dashes under utf-8. (closes: #159907)
diff -Nur ../lockfile-progs-0.1.10/debian/control ./debian/control
--- ../lockfile-progs-0.1.10/debian/control	2004-02-28 10:27:12.000000000 -0700
+++ ./debian/control	2005-09-30 16:29:12.000000000 -0600
@@ -3,14 +3,13 @@
 Priority: optional
 Maintainer: Rob Browning <rlb@defaultvalue.org>
 Build-Depends: liblockfile-dev
-Standards-Version: 3.6.1
+Standards-Version: 3.6.2
 
 Package: lockfile-progs
 Section: misc
 Priority: optional
 Architecture: any
 Depends: ${shlibs:Depends}
-Priority: optional
 Description: Programs for locking and unlocking files and mailboxes
  This package includes several programs to safely lock and unlock
  files and mailboxes from the command line.  These include:
diff -Nur ../lockfile-progs-0.1.10/debian/shlibs.local ./debian/shlibs.local
--- ../lockfile-progs-0.1.10/debian/shlibs.local	1969-12-31 17:00:00.000000000 -0700
+++ ./debian/shlibs.local	2005-09-30 14:38:26.000000000 -0600
@@ -0,0 +1 @@
+liblockfile 1   liblockfile1 (>= 1.04)
diff -Nur ../lockfile-progs-0.1.10/lockfile-progs.1 ./lockfile-progs.1
--- ../lockfile-progs-0.1.10/lockfile-progs.1	2004-02-28 10:44:05.000000000 -0700
+++ ./lockfile-progs.1	2005-09-30 15:02:00.000000000 -0600
@@ -8,7 +8,7 @@
 .\" License as specified in the file COPYING that comes with the
 .\" lockfile\-progs distribution.
 .\"
-.TH lockfile\-progs 1 "23 April 2001" "0.1.10" "Lockfile programs"
+.TH lockfile\-progs 1 "30 September 2005" "0.1.11" "Lockfile programs"
 .SH NAME
 lockfile\-progs \- command\-line programs to safely lock and unlock
 files and mailboxes (via liblockfile).
@@ -92,9 +92,7 @@
 5 second increments) until a maximum delay of one minute between
 retries is reached.  The default retry count is 9 which, if all 9
 attempts to lock the file fail, will give up after 180 seconds (3
-minutes).  Note that the description above matches the one for the
-underlying lockfile_create(3) call, but in fact, this option might be
-better named \-\-tries since \-\-retry 0 will not try to lock at all.
+minutes).
 
 .SH EXAMPLES
 
